lib/ansible/modules/network/ovs/openvswitch_bond.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2019, James Denton <james.denton@outlook.com>
+# Portions copyright @ 2013 David Stygstra <david.stygstra@gmail.com>
+# Portions copyright @ 2015 VMware, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: openvswitch_bond
+version_added: 2.8
+author: "James Denton (@busterswt)"
+short_description: Manage Open vSwitch bonds
+requirements: [ ovs-vsctl ]
+description:
+    - Manage Open vSwitch bonds and associated options.
+options:
+    bridge:
+        required: true
+        description:
+            - Name of bridge to manage
+    port:
+        required: true
+        description:
+            - Name of port to manage on the bridge
+    interfaces:
+        required: true
+        type: list
+        description:
+            - List of interfaces to add to the bond
+    bond_mode:
+        choices: [ active-backup, balance-tcp, balance-slb ]
+        description:
+            - Sets the bond mode
+    lacp:
+        choices: [ active, passive, off ]
+        description:
+            - Sets LACP mode
+    bond_updelay:
+        description:
+            - Number of milliseconds a link must be up to be activated
+              to prevent flapping.
+    bond_downdelay:
+        description:
+            - Number of milliseconds a link must be down to be deactivated
+              to prevent flapping.
+    state:
+        default: "present"
+        choices: [ present, absent ]
+        description:
+            - Whether the port should exist
+    timeout:
+        default: 5
+        description:
+            - How long to wait for ovs-vswitchd to respond
+    external_ids:
+        default: {}
+        description:
+            - Dictionary of external_ids applied to a port.
+    other_config:
+        default: {}
+        description:
+            - Dictionary of other_config applied to a port.
+    set:
+        type: list
+        description:
+            - Sets one or more properties on a port.
+'''
+
+EXAMPLES = '''
+# Creates an active-backup bond using eth4 and eth5 on bridge br-ex
```

Comment:
We suggest that people always use a `- name:` when writing Playbooks, doing the same in the EXAMPLES makes it easier for people to copy&paste:
```suggestion
- name: Creates an active-backup bond using eth4 and eth5 on bridge br-ex
```